ironfish/src/rpc/routes/wallet/getPublicKey.ts

```diff
@@ -1,14 +1,15 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Address } from '@ethereumjs/util'
 import * as yup from 'yup'
 import { ApiNamespace } from '../namespaces'
 import { routes } from '../router'
 import { AssertHasRpcContext } from '../rpcContext'
 import { getAccount } from './utils'
 
 export type GetPublicKeyRequest = { account?: string }
-export type GetPublicKeyResponse = { account: string; publicKey: string }
+export type GetPublicKeyResponse = { account: string; publicKey: string; evmPublicKey?: string }
```

Comment:
Could we call this `evmPublicAddress`? The address is the last 20 bytes of a hash of the public key. The public key isn't really used for anything

Reply:
Good call